highfidelity/hifi
# Bug fix for Users unable to Squat

This pr addresses case https://highfidelity.manuscript.com/f/cases/17996/Avatar-can-not-stay-in-a-squatted-position-while-in-VR

1 Created a new SitStandStateType  Force Standing.  When this is enabled then the user can squat down to the ground without vertical recentering being triggered.  At the same time they are still able to walk around in room scale in order to move about the room.

### Test Plan
1) start interface. 
2) go to settings controls and set the Avatar Leaning Behavior to 3. Standing
3) go into hmd mode and sit down. then go down on one knee.
4) expected result: there should be no vertical recentering. allowing users to sit on the ground with out causing recentering.
